src/server/api/routers/webhook.ts

```diff
@@ -0,0 +1,50 @@
+import { Octokit } from "@octokit/rest";
+import { createTRPCRouter, protectedProcedure } from "~/server/api/trpc";
+
+export const webhookRouter = createTRPCRouter({
+  createWebhook: protectedProcedure.mutation(async () => {
+    const owner = "GabrielPedroza";
+    const repo = "e-commerce";
+
+    const WEBHOOK_URL = "https://bread-meta.vercel.app/api/webhooks/webhook";
```

Comment:
this is the deployed URL, how would you test changes to this router locally?

Reply:
I don't think there is a way ti test it locally. GitHub requires a public url in order to send alerts and I don't have laptop permission to use ngrok or tl tunneling